qemu/tests/cfg/hotplug_mem_share_discard_data.cfg

```diff
@@ -0,0 +1,38 @@
+- hotplug_mem_share_discard_data:
+    only Linux
+    type = hotplug_mem_share_discard_data
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.10
+    start_vm = no
+    not_preprocess = yes
+    prealloc_mem = yes
+    slots_mem = 4
+    maxmem_mem = 32G
+    mem = 4096
```

Comment:
To use fixed mem for any env ?   If so, suggest `mem_fixed`

Reply:
Changed!